source/vstd/arithmetic/power2.rs

```diff
@@ -239,4 +237,168 @@ pub proof fn lemma2_to64()
     ) by(compute_only);
 }
 
+/// Mask with low n bits set.
+pub open spec fn mask(n: nat) -> nat {
+    (pow2(n) - 1) as nat
+}
+
+/// Proof relating the n-bit mask to a function of the (n-1)-bit mask, for given
+/// n.
+pub proof fn lemma_mask_unfold(n: nat)
```

Comment:
Same question about "unfold" terminology here.

Reply:
unfold seems fine to me.